data/json/vehicleparts/vp_flags.json

```
@@ -54,6 +54,13 @@
     "context": [ "vehicle_part" ],
     "info": "With a seat or saddle, you drive from here.  You can 'e'xamine the tile to access the controls, or use the vehicle control key (default '^')."
   },
+  {
+    "id": "CTRL_WO_HANDS",
```

```suggestion
    "id": "CONTROL_WITHOUT_HANDS",
```

avoid using acronyms unless needed.